sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -26,10 +26,12 @@ fe2o3-amqp-management = { workspace = true, optional = true }
 fe2o3-amqp-types = { workspace = true, optional = true }
 serde_amqp = { workspace = true, optional = true }
 serde_bytes = { workspace = true, optional = true }
+typespec = { workspace = true, features = ["amqp_errors"] }
```

Comment:
I don't think we need to specialize on "errors". Maybe just call the feature "amqp". We have "http" now. I don't want feature to be so granular that you feel like you're ordering off a Cheesecake Factory menu.